example/manifests/observatorium.yaml

```diff
@@ -124,9 +121,6 @@ spec:
         resources:
           requests:
             storage: 50Gi
-  securityContext:
-    fsGroup: 65534
-    runAsUser: 65534
```

Comment:
I am wondering what are the ramifications of removing `securityContext`. It seems that `fsGroup` cannot be defined in `v1.SecurityContext ` on container level, but this field is available on pod level in `v1.PodSecurityContext`.